app/admin/topics.rb

```diff
@@ -0,0 +1,36 @@
+ActiveAdmin.register Topic do
+  permit_params :name, :icon
+
+  form do |f|
+    f.inputs 'Details' do
+      f.input :name, required: true
+      f.input :icon, as: :file
+    end
+
+    actions
+  end
+
+  index do
+    selectable_column
+    id_column
+    column :name
+    column :icon do |topic|
+      image_tag topic.icon.url, class: 'thumbnail' if topic.icon.present?
```

Comment:
Use `topic.icon_url`, it's a carrierwave method added to the model, this way you don't need to check if the icon is present

Reply:
@GuilleLeopold I still need to check for presence, otherwise it will crash because of nil param. Based on carriewave Readme examples and of course, actual testing of the change.